src/features/smartSearch/components/filters/Journey/DisplayJourney.tsx

```diff
@@ -80,33 +73,21 @@ const DisplayJourney: FC<DisplayJourneyProps> = ({ filter }): JSX.Element => {
     }
   };
 
+  const state = filter.config.closed ? 'closed' : 'open';
```

Comment:
I thought this could possibly be a enum value, if that makes sense for this use case, since I imagine a lot of places would use these two words. But I am also very much a fan of enums since I easily make spelling mistakes in strings... 
So i guess more of a question than a comment - would an enum make sense here?

Reply:
I think an enum is overkill here, and I will try to motivate why (because it's mostly a gut feeling, so I need to really introspect to be able to motivate it).

Enums, to me, make sense when something needs to be _formalized_, e.g. as part of a contract/API, i.e. when the value is important not just for the code right here, but also for other parts of the code. The enum is good in those cases because it gives the concept a _name_ that will appear whenever the enum is used (the name of the enum), which a simple string union type wouldn't.

In this case, the `state` is just a local variable to avoid duplicating the logic. It could have been a boolean called `isClosed` instead, but I chose this way because of similarly gutty feelings around booleans. 😊 

You mention spelling mistakes, and that won't be a problem here. The type of `state` is `'closed' | 'open'`, as inferred by TypeScript, so it's impossible to use any other value elsewhere. Of course it could be possible to misspell it here and then _again_ wherever it's used, but then I would argue that is not a big problem (and that problem exists for enums as well).

Reply:
Thank you for your thoughts! good for learning :)